features/0592-indy-attachments/README.md

```diff
@@ -154,7 +211,7 @@ Here is a sample proof request that embodies the following: "Using a government-
 
 This is the format of an Indy-style ZKP. It plays the same role as a W3C-style verifiable presentation (VP) and can be [mapped to one](https://docs.google.com/document/d/1ntLZGMah8iJ_TWQdbrNNW9OVwPbIWkkCMiid7Be1PrA/edit#heading=h.vw0mboesl528).
 
-This is the format emitted by libindy's [indy_prover_create_proof()](https://github.com/hyperledger/indy-sdk/blob/57dcdae74164d1c7aa06f2cccecaae121cefac25/libindy/src/api/anoncreds.rs#L1404)) function. A proof that responds to the [previous proof request sample](#proof-request-format) looks like this:
+The identifier for this format is `hlindy@v2.0` in the context of a Presentation message. It is the base64-encoded version of the data emitted by libindy's [indy_prover_create_proof()](https://github.com/hyperledger/indy-sdk/blob/57dcdae74164d1c7aa06f2cccecaae121cefac25/libindy/src/api/anoncreds.rs#L1404)) function. A (non-base64-encoded) proof that responds to the [previous proof request sample](#proof-request-format) looks like this:
```

Comment:
indy_prover_create_proof already emits data in base64-encoded data. The verbiage here implies that the encoding happens after the fact. I suggest rewording to "It is the data emitted by... function (which emits the data base64-encoded). A (non-base64-encoded)...

Reply:
indy_prover_create_proof emits JSON-encoded, not base64-encoded data.

Reply:
Sorry, I was confusing this with a different function that emits base64-encoded data. You are correct and I am not; the data is JSON as emitted from the function.

The rationale for special-casing JSON encoding for an attachment was to make data easier to read for developers in the body of a message. If we're not using *embedded* attachments, I think base64 should be used, as this argument is weaker. Base64 is also required if the attachment will be signed (to avoid the JSON canonicalization problem). And DIDComm v2 always uses base64. None of these observations means that you are incorrect, though; any form of attachment ought to work.